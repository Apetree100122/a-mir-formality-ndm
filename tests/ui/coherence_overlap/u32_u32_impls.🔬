[
    crate core {
        trait Foo<> where [] {}
        impl<> Foo<> for u32 where [] {}
        impl<> Foo<> for u32 where [] {}
    }
]
