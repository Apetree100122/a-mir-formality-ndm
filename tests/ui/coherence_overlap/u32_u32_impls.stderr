Error: duplicate impl in current crate: impl Foo for u32 where [] { }
